docs/content/docs/updates/v1.0.0.md

```diff
@@ -9,8 +9,8 @@ summary: |
 ## Patch Updates
 
 ### Update 1.0.1:
-- Add the missing `ConnectionState` export to `fluid-framework`
-- Fix AzureClient issue where the second user could not connect in `local` mode
+- Add the missing [ConnectionState](https://fluidframework.com/docs/apis/container-loader#connectionstate-Enum) export to `fluid-framework`
+- Fix [AzureClient](https://fluidframework.com/docs/apis/azure-client/azureclient) issue where the second user could not connect in `local` mode
```

Comment:
Can you use "Hugo links" like `relref`? They're a lot more resilient to site reorganization.

Reply:
Yes, can do